luigi/execution_summary.py

```diff
@@ -270,15 +277,15 @@ def _get_comments(group_tasks):
     ("already_done", 'present dependencies were encountered'),
     ("completed", 'ran successfully'),
     ("failed", 'failed'),
-    ("scheduling_error", 'failed running complete() or requires()'),
+    ("scheduling_error", 'failed scheduling'),
     ("still_pending", 'were left pending, among these'),
     ("still_pending_ext", 'were missing external dependencies'),
     ("run_by_other_worker", 'were being run by another worker'),
     ("upstream_failure", 'had failed dependencies'),
     ("upstream_missing_dependency", 'had missing external dependencies'),
     ("upstream_run_by_other_worker", 'had dependencies that were being run by other worker'),
-    ("upstream_scheduling_error", 'had dependencies whose complete() or requires() failed'),
-    ("unknown_reason", 'were left pending because of unknown reason'),
+    ("upstream_scheduling_error", 'had dependencies whose scheduling failed'),
+    ("unknown_reason", 'did not run successfully because of unknown reason'),
```

Comment:
Mhm, it stems from [this part of discussion](https://github.com/spotify/luigi/pull/1771#discussion_r71123666). Thinking about the cases of disabling and resources missing, it's still true that the instance remained "pending". (In the common sense common denominator of those more detailed statuses. If we invested more in status propagation we would come up with summaries like "pending because missing resources" etc.)

I'm saying that in my view the higher level conceptual task's flow hasn't changed much and doesn't risk changing much. It's first pending, then becomes running (the moment when `run()` is invoked). The reason for failing running is always "known" via the exception; the reason for not running at all can vary, but "left pending" remains valid wording.